metricflow/model/transformations/dbt_to_metricflow.py

```diff
@@ -202,7 +202,6 @@ def build_data_source_for_metric(self, dbt_metric: DbtMetric) -> DataSource:
             name=metric_model_ref.name,
             description=metric_model_ref.description,
             sql_table=data_source_table,
-            dbt_model=data_source_table,
```

Comment:
This seems fine to me but I don't know if this affects the conversion tooling @callum-mcdata and @Jstein77 are working on.

If it does it's probably ok - they can fork from an earlier point in the source tree - but just wanted to mention it.

Reply:
I'm not aware of the conversion tooling they are working on there, but wanted to point out that the only use case at the moment for the `dbt_model` property was that it is used by the Transform UI purely for showing lineage metadata. So hopefully there's some work around there if they are using it.